recipe/conda_build_config.yaml

```diff
@@ -1,14 +1,12 @@
 # cuda 9.2, gcc 7.3, cudnn 7.6
 # cuda 10.0, gcc 7.3, cudnn 7.6
 # cuda 10.1, gcc 7.3, cudnn 7.6
-c_compiler_version:           # [linux]
+c_compiler_version:           # [unix]
   - 7.3                       # [linux]
-c_compiler_version:           # [osx]
-  - 4                        # [osx]
-cxx_compiler_version:         # [linux]
+  - 4                         # [osx]
+cxx_compiler_version:         # [unix]
   - 7.3                       # [linux]
-cxx_compiler_version:         # [osx]
-  - 4                        # [osx]
+  - 4                         # [osx]
 cross_compiler_target_platform:   # [win]
   - win-64                        # [win]
```

Comment:
is this required? I suspect it.  at least it looks pretty strange to describe here cross for win-64

Reply:
This comment is based on an older version of the PR and is not applicable anymore.